readalongs/cli.py

```diff
@@ -29,13 +24,11 @@
 from tempfile import TemporaryFile
 
 import click
-from flask.cli import FlaskGroup
```

Comment:
Does this change mean we're actually no longer supporting your old web page app at all anymore? Or is using @cli.command vs @app.cli.command still compatible with views.py?

Reply:
No, just tested, it does not. Nice to remove unneeded imports for sure, especially when they only add stuff we never use (anymore, anyway).

Reply:
you can always run it with `python3 run.py` - I'll remove the app import and create_app function too